BackEnd/administration/views.py

```diff
@@ -122,3 +123,18 @@ class AutoModerationHoursView(RetrieveUpdateAPIView):
 
     def get_object(self):
         return AutoModeration.get_auto_moderation_hours()
+
+
+class ModerationEmailView(RetrieveUpdateAPIView):
+    """
+    View for retrieving and updating the ModerationEmail instance.
+    Requires the user to be a superuser.
+    """
+
+    permission_classes = [IsSuperUser]
+
+    queryset = ModerationEmail.objects.all()
```

Comment:
This line with queryset is redundant. If you override get_object() the way you did, setting queryset simply does nothing.

Reply:
fixed it, thanks